<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelAgencyUI</class>
 <widget class="QMainWindow" name="TravelAgencyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TravelAgencyUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Kund*in</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditID"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Reisen</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>138</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetReisen">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>Reise-ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Beginn der Reise</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ende der Reise</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxReise">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>611</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Reise</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>591</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Reise-ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditReiseID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Buchungen</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>Buchungstyp</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Start</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ende</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Preis</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionDatei_Anlegen"/>
    <addaction name="actionSearch"/>
    <addaction name="actionSpeichern"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDatei_Anlegen"/>
   <addaction name="actionSearch"/>
   <addaction name="actionSpeichern"/>
  </widget>
  <action name="actionDatei_Anlegen">
   <property name="icon">
    <iconset resource="Graphics.qrc">
     <normaloff>:/icons/import_icon.png</normaloff>:/icons/import_icon.png</iconset>
   </property>
   <property name="text">
    <string>Datei Anlegen</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="Graphics.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset resource="Graphics.qrc">
     <normaloff>:/icons/pngtree-vector-save-icon-png-image_322173.jpg</normaloff>:/icons/pngtree-vector-save-icon-png-image_322173.jpg</iconset>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Graphics.qrc"/>
 </resources>
 <connections/>
</ui>
